ui/components/forms/flavors/compound-form/_index.scss

```diff
@@ -21,7 +21,10 @@
   }
 
   .slds-form-element {
-    width: 100%; // Default - Stretch 100% unless told otherwise using sizing helpers
+    // Default - Stretch 100% unless told otherwise using sizing helpers
+    &:not([class*="slds-size--"]) {
+      width: 100%;
+    }
```

Comment:
What about responsive sizes? `.slds-medium-size`, `.slds-small-size`...

Ideally the `slds-size--` classes should have a higher specificity so we wont need to do this. 

Is it possible to increase the specificity of `slds-size--` or decrease this ones? Otherwise we will end up having to add this hacks everywhere.